modules/scripts/startup-script/main.tf

```diff
@@ -26,11 +26,15 @@ locals {
     destination = "install_cloud_ops_agent_automatic.sh"
   }] : []
 
-  ssh_args = join("", [
-    "-e host_name_prefix=${var.deployment_name}"
-  ])
+  configure_ssh = length(var.configure_ssh_host_prefix) > 0
+  host_args = {
+    host_name_prefix = var.configure_ssh_host_prefix
+  }
+  #  host_args = "{host_name_prefix:${jsonencode(var.configure_ssh_host_prefix)}}"
+
+  ssh_args = "-e  @/tmp/prefix_file.json"
```

Comment:
nit: This local is not really earning its keep. You could make the local just `/tmp/prefix_file.json` and then use it on line 51 and 57 to avoid specifying the path twice.

Reply:
good point, I restructured the code.